### ISSUE TITLE
Question regarding --count

### ISSUE THREAD

**Issue by milliorn** [instaloader/instaloader#2140]
https://instaloader.github.io/cli-options.html?highlight=count#cmdoption-count

I tried to use COUNT yesterday. I'm unsure if I incorrectly used it, misunderstood the documentation, or maybe there is an issue? When I used it, it did not stop at 100, and keep going, until I manually stopped it after it hit 300.

Inside command line, I used this below.
`
instaloader.exe profiler milliorn --filename-pattern={filename} --latest-stamps --no-videos --no-video-thumbnails --no-captions --no-metadata-json --no-compress-json --no-profile-pic --max-connection-attempts=1 --abort-on=302,400,401,429 --latest-stamps ./latest-stamps.ini --user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36" --count=100
`

The reason I added it to the end was to test it out. Everything before it works. I have been successful with this (before using --count) in WSL, and Windows Command Prompt (command is different in WSL, yet works just fine). 

What am I doing wrong, or what am I misunderstanding here?

**Comment by oliverjrose99** [comment]
the format should be `-count 100` or `-c 100`
note the space and no = symbol

**Comment by milliorn** [comment]
> the format should be `-count 100` or `-c 100` note the space and no = symbol

I tried with `--count 1` and `-c 1` but that didn't work.  Then I tried with `--count 10` and `-c 10` but that didn't work and in both attempts I had to manually stop it. I am using Windows Terminal Version: 1.18.3181.0 Command Prompt.

**Comment by Showmaster3000** [comment]
please read the description of count until the end:
`Applies to #hashtag, %location_id, :feed, and :saved.`
Your command line doesn't contain anything of that. count does NOT work for posts or stories.

**Comment by milliorn** [comment]
> please read the description of count until the end: `Applies to #hashtag, %location_id, :feed, and :saved.` Your command line doesn't contain anything of that. count does NOT work for posts or stories.

Could this become a feature request?

**Comment by Showmaster3000** [comment]
> Could this become a feature request?

**Comment by milliorn** [comment]
I will say thank you for your help. After reading that link, I don't think I want to have any more discussions here unless I absolutely have too. Telling someone to read the description, when I obviously did, and obviously I did my due diligence to read the documentation else I would not have been able to run any of those commands and flags. If you meant to say I made a mistake in what I read, then yes that is correct, and would explain why I hit a wall. But to just fire off "read the description" as if I not, lazy, etc. is not appropriate behavior. 

Regardless, I'll close this and say thank you again.
